server/db/testDB/build.sql

```diff
@@ -0,0 +1,29 @@
+BEGIN;
+
+DROP TABLE IF EXISTS salesforce.application__c, salesforce.questions__c CASCADE;
+
+CREATE TABLE salesforce.application__c (
+id SERIAL PRIMARY KEY,
+question_1__c TEXT NOT NULL,
+question_2__c TEXT NOT NULL, 
+question_3__c TEXT NOT NULL
+
+);
+
+CREATE TABLE salesforce.questions__c (
+id SERIAL PRIMARY KEY,
+title__c TEXT NOT NULL, 
+wordsLimit__c INT NOT NULL, 
+tips__c TEXT, 
+expectations__c TEXT
+
+);
+
+INSERT INTO salesforce.application__c (question_1__c,  question_2__c  ,question_3__c ) VALUES 
```

Comment:
you can split this code in another file one for the tables and the other for the fake data you have

Reply:
I'm having issues with the running order when I'm splitting it up. I think I've to create it as two promises and resolve it with a promise.all(), is there a way I'm missing?